packages/vaadin-messages/src/vaadin-message-input-text-area.js

```diff
@@ -41,34 +45,53 @@ class MessageInputTextAreaElement extends TextAreaElement {
     };
   }
 
-  ready() {
-    super.ready();
+  /** @protected */
+  connectedCallback() {
+    super.connectedCallback();
 
-    const textarea = this.inputElement;
-    textarea.removeAttribute('aria-labelledby');
+    if (this.inputElement) {
+      this.inputElement.removeAttribute('aria-labelledby');
+      this.__updateAriaLabel(this.ariaLabel);
 
-    // Set initial height to one row
-    textarea.setAttribute('rows', 1);
-    textarea.style.minHeight = '0';
+      // Set initial height to one row
+      this.inputElement.setAttribute('rows', 1);
+    }
```

Comment:
**nit:** Shouldn't this part be in the `_inputElementChanged` method instead?

Reply:
Thanks, done.